### DISCUSSION TITLE
websocket setup and use instructions

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#383] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I have tried to use the WebSocket wifi connection, but I just cannot seem to get it to work.

**Describe the solution you'd like**
I would like a clear and concise description of how to set up and use the WebSockets on the website.

**Comment by yume-chan** [comment]
What steps did you do and in which step it has problem?

**Reply by CorruptionVR** [reply]
the problem is that I don't know how to enable WebSockets on my device. I have tried to enable wifi debugging in the website, setting the port, and then trying to connect my device to the website with my wifi debugging address and port.
maybe I'm just stupid and need to do something else.

**Reply by CorruptionVR** [reply]
also, I am doing this on a stable, non-dev mode chrome OS

**Comment by yume-chan** [comment, marked as answer]
The old instruction is at https://github.com/yume-chan/ya-webadb/discussions/245#discussioncomment-384030

It's a little different from current UI, now you need to manually add the WebSocket backend from the `Add` split button, using `ws://localhost:15555` as URL.
